migrations/sqls/20220224164015-create-note-table-up.sql

```diff
@@ -0,0 +1,20 @@
+drop table if exists water.notes;
```

Comment:
doesn't make any difference if this is here or not so we can remove it if you want